supertokens/supertokens.go

```diff
@@ -31,6 +31,8 @@ type superTokens struct {
 	RecipeModules         []RecipeModule
 	OnSuperTokensAPIError func(err error, req *http.Request, res http.ResponseWriter)
 	Telemetry             *bool
+	Log                   Logger
+	RequestIDKey          any
 }
```

Comment:
this works, can you add test(s) to this PR for the changes you have made?

Reply:
Added some tests for the `config` changes as well as the `utils` function to extract the `requestID`